src/js/uio.js

```diff
@@ -0,0 +1,49 @@
+/* global $, fluid */
+
+$(document).ready(function () {
+	fluid.defaults("fluid.prefs.schemas.lineSpace", {
+		gradeNames: ["fluid.prefs.schemas"],
+		schema: {
+			"fluid.prefs.lineSpace": {
+				"type": "number",
+				"default": 1,
+				"minimum": 1,
+				"maximum": 2,
+				"multipleOf": 0.1
+			}
+		}
+	});
```

Comment:
@cindyli this is interesting. You redefined the `fluid.prefs.schemas.lineSpace` schema grade from Infusion. I think we might need to ask @amb26 about this approach.

Reply:
This approach is dubious and should be replaced with an options distribution, grade linkage, or other such thing - otherwise, please issue a pull against Infusion!

Reply:
@amb26, I'm not completely sure, but I suspect that the issue may revolve around the difficulty of specifying an alternative range for the preference in a particular instance. These values default to what comes from the primary schema, but may not be reflective of what is desired in a particular instance of a UIO/Prefs Editor. @cindyli could you please confirm?

Reply:
@amb26, as @jobara said, it's not easy to use `distributeOptions` or grade linkage to override values that come from the primary schema.

The primary builder uses [a function](https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/PrimaryBuilder.js#L54) to look up all primary schema grades registered with the prefs framework using the base grade `fluid.prefs.schemas`. The returned value of this function is then manipulated by [`{that}.buildPrimary`](https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/PrimaryBuilder.js#L48) that returns an array of a set of different grade names that are then attached onto the component tree.

This means, even though the value i wanna change is from a primary schema grade named `fluid.prefs.schemas.lineSpace`, this grade is not on the component tree after going thru the primary builder process.

I think this leaves me with 2 options:
1. The easy way is to override `fluid.prefs.schemas.lineSpace` directly so the primary builder will read my version;
2. The difficult way is to dig into the code to figure out how `fluid.prefs.schemas.lineSpace`  gets transformed and where the value I wanna change is ended up on the component tree. Then use `distributeOptions` to override this value.

I lean towards using the easy way for now and file a JIRA against the prefs framework for an improvement.

@amb26, @jobara, what do you think?